docs/Tutorial/getting-started.md

```diff
@@ -0,0 +1,128 @@
+# Quick guide
+
+## What you'll do
+
+- Deploy the nginx-ingress-integrator charm
+- Relate it to another charm
+- Inspect the ingress it creates.
+
+We'll then also look at changing that ingress via a juju configuration update.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
```

Comment:
I see most of the time is `you` but I see some `we`s too, should it be consistent?